pkgs/top-level/all-packages.nix

```diff
@@ -28861,6 +28861,17 @@ with pkgs;
 
   picom = callPackage ../applications/window-managers/picom {};
 
+  picom-next = picom.overrideAttrs (oldAttrs: rec {
+    version="vNext";
```

Comment:
```suggestion
      version = "vNext";
```

Reply:
Not sure if we want it to have a proper version instead.

Reply:
If so? which version
vNext indicates its the updated next branch. Idk either

Reply:
Not sure too, you can leave it as-is for now. Maybe some other reviewer has a better idea.